suites/pacific/upgrades/tier-2_upgrade_test-multi-path-upgrade-to-5.2-latest.yaml

```diff
@@ -0,0 +1,100 @@
+# Upgrade: Multihop upgrade evaluation
+#
+# Conf: conf/pacific/upgrades/5-0_upgrades.yaml
+#
+# This test suite is used to test upgrades from all previous ceph release versions
+# to the latest 5.x version. We start from RHCS 4 and install a CDN cluster
+# then proceed with upgrading to every released version after that
+# finally upgrading the cluster to the latest 5.x version selected from the CLI
+# using the configuration given in the suite
+#
+
+# RHCS 5.x multihop upgrade test suite.
+tests:
+- test:
+    name: install ceph pre-requisites
+    module: install_prereq.py
+    abort-on-fail: True
+
+- test:
+    name: upgrade the RHCS cluster based on the path specified
+    polarion-id: ""
+    module: test_upgrade.py
+    config:
+      paths:
+        4.2:
+          config:
+            ansi_config:
+              osd_scenario: lvm
+              osd_auto_discovery: False
+              copy_admin_key: true
+              dashboard_enabled: True
+              dashboard_admin_user: admin
+              dashboard_admin_password: p@ssw0rd
+              grafana_admin_user: admin
+              grafana_admin_password: p@ssw0rd
+              cephfs_pools:
+                - name: "cephfs_data"
+                  pgs: "8"
+                - name: "cephfs_metadata"
+                  pgs: "8"
```

Comment:
I would like us to revisit this setting.

Reply:
Done.